website/src/docs/hotchocolate/filtering/filter-naming-convention.md

```diff
@@ -0,0 +1,158 @@
+# Naming Conventions
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.